wasm/silkworm_wasm_api.cpp

```diff
@@ -49,7 +49,7 @@ intx::uint256* new_uint256_le(uint64_t a, uint64_t b, uint64_t c, uint64_t d) {
 
 void delete_uint256(intx::uint256* x) { delete x; }
 
-const ChainConfig* lookup_config(uint64_t chain_id) { return lookup_chain_config(chain_id); }
+const ChainConfig* lookup_config(uint64_t chain_id) { return lookup_known_chain(chain_id)->second; }
```

Comment:
`lookup_known_chain` can return `std::nullopt`, so I suggest to change this function body to:
```
const auto chain{lookup_known_chain(chain_id)};
if (!chain) {
    return nullptr;
}
return chain->second;
```